### PR TITLE
added dtype support for brighttemp, removed raster_tester test dep

### LINKED ISSUE #22: brighttemp dst-dtype scaling error
```
$ rio toa brighttemp -j 1 -t 'LC8.*\_B{b}.tif' --dst-dtype uint16 \
    LC80130312015295LGN00_B11.tif \
    LC80130312015295LGN00_MTL.json \
    LC80130312015295LGN00_B11_toa.tif
```

gives me an error related to array dtype. Probably just needs to be cast to the dst dtype? No scaling necessary since they are absolute units

```
Traceback (most recent call last):
...
  File "/Users/mperry/work/rio-mucho/riomucho/__init__.py", line 110, in run
    dst.write(data, window=window)
  File "rasterio/_io.pyx", line 1616, in rasterio._io.DatasetWriterBase.write (rasterio/_io.c:23593)
ValueError: the array's dtype 'float32' does not match the file's dtype 'uint16'
```

### LINKED ISSUE #25: remove raster_tester dependency
It's not yet compatible with python 3 or rasterio 1.0 and it's not a public repo. Let's axe it and just vendorize the two functions.

### PR DESCRIPTION
**Pull request by perrygeo**
Resolves #25 and Resolves #22 

A few other minor tweaks, explained in comments inline.

Ready for review, @virginiayung

### CODE REVIEW

File: rio_toa/scripts/cli.py [hunk 1]
```diff
@@ -105,7 +105,7 @@ def reflectance(ctx, src_paths, src_mtl, dst_path, dst_dtype,
               type=click.Choice(['float32', 'float64', 'uint16', 'uint8']),
               default='float32',
               help='Output data type')
-@click.option('--temp_scale', '-s',
+@click.option('--temp-scale', '-s',
```
Comment (perrygeo) [inline rio_toa/scripts/cli.py:108]: Using  `-` for word delimiters in command line options.

File: tests/test_brightness_temp.py [hunk 2]
```diff
@@ -1,19 +1,37 @@
...
-import riomucho
-import pytest
 from hypothesis import given
 import hypothesis.strategies as st
 from hypothesis.extra.numpy import arrays
-from rasterio.coords import BoundingBox
+import numpy as np
+import pytest
+import rasterio as rio
+from rasterio.warp import reproject, Resampling
```
Comment (perrygeo) [inline tests/test_brightness_temp.py:13]: Reorganized imports into three sections per Rasterio's convention:
- standard lib imports (alphabetized)
- third-party imports (alpha)
- project-level imports

File: tests/test_reflectance.py [hunk 3]
```diff
@@ -1,16 +1,37 @@
...
+    reproject(
+        bidx, upBidx,
+        src_transform=fr,
+        dst_transform=to,
+        src_crs="EPSG:3857",
+        dst_crs="EPSG:3857",
+        resampling=Resampling.bilinear)
+
+    return upBidx
+
```
Comment (perrygeo) [inline tests/test_reflectance.py:33]: Inlined the raster_tester functions to remove the implicit dependency

**Comment by virginiayung** [comment]
Looks great @perrygeo! Thanks for making these changes.

**Pull request merged**